include/mbgl/util/compression.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <string>
+
+namespace mbgl {
+    namespace util {
+        
+        std::string compress(const std::string &raw);
+        std::string decompress(const std::string &raw);
```

Comment:
Nit: the convention inside mbgl is to place `&` and `*` next to the type rather than the symbol name, so `const std::string& raw`. This is different than the convention in our Objective-C++ code at the SDK level.